tests/fuzz/Readme.md

```diff
@@ -1,7 +1,28 @@
 # Fuzz Tests
-By default, every test in this directory will be run as a fuzz test for several minutes each during builds. To run all fuzz tests simply run `make fuzz` from the top `s2n` directory to compile s2n with the proper flags and run the fuzz tests. To run a specific subset of fuzz tests, simply set the FUZZ_TESTS variable as follows:
-
-> FUZZ_TESTS="test1 test2 test3"
+By default, every test in this directory will be run as a fuzz test for several minutes each during builds. To run all fuzz tests, run the following commands from the top `s2n-tls` directory:
+1. Compile with S2N_FUZZ_TEST option enabled
+```
+cmake . -Bbuild \
+-DCMAKE_PREFIX_PATH=/usr/local/$S2N_LIBCRYPTO \
```

Comment:
We should try and avoid CI-specific paths in public documentation. Also CMake should be able to find the libcrypto automatically.
```suggestion
```